internal/services/vorpalengine/vorpal.go

```diff
@@ -0,0 +1,193 @@
+package vorpalengine
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/checkmarx/ast-cli/internal/commands/vorpal/vorpalconfig"
+	errorconstants "github.com/checkmarx/ast-cli/internal/constants/errors"
+	"github.com/checkmarx/ast-cli/internal/logger"
+	"github.com/checkmarx/ast-cli/internal/params"
+	"github.com/checkmarx/ast-cli/internal/services/osinstaller"
+	"github.com/checkmarx/ast-cli/internal/wrappers"
+	"github.com/checkmarx/ast-cli/internal/wrappers/grpcs"
+	getport "github.com/jsumners/go-getport"
+	"github.com/spf13/viper"
+)
+
+type VorpalScanParams struct {
```

Comment:
suggestion
it better to save the convention we write now - 
the wrappers have to be send as params